packages/db/src/schema/core/defs.ts

```diff
@@ -274,6 +304,13 @@ export const competitions = pgTable(
     }),
     status: competitionStatus("status").notNull(),
     sandboxMode: boolean("sandbox_mode").notNull().default(false),
+    engineId: text("engine_id"),
+    engineVersion: text("engine_version"),
+    engineConfig: jsonb("engine_config"),
+    participationConfig: jsonb("participation_config"),
+    partners: jsonb("partners"),
+    rewards: jsonb("rewards"),
```

Comment:
I was going to say this should be a table or additional columns here, but it is described as "hints... display-only", so maybe we want to maintain the flexibility provided by json here.